au4/src/projectscene/view/clipsview/selectionviewcontroller.cpp

```diff
@@ -107,6 +107,19 @@ void SelectionViewController::onSelectionDraged(double x1, double x2, bool compl
     selectionController()->setDataSelectedEndTime(m_context->positionToTime(x2, true /*withSnap*/), completed);
 }
 
+void SelectionViewController::selectTrackAudioData(double y)
+{
+    const std::vector<TrackId> tracks = determinateTracks(m_startPoint.y(), y);
+    selectionController()->setSelectedTrackAudioData(tracks.at(0));
```

Comment:
can we replace this with `selectionController()->setSelectedTrack(trackId)`?

Reply:
not quite: right here we want to precisely select all of the audio data that belongs to the track. The function you mention just marks that currently track with particular trackId was clicked for example (or that you selected part of that data, etc.)